docs/clients/power_safety.rst

```diff
@@ -0,0 +1,144 @@
+
+Power Safety
+-------------
+
+The Power Safety protects the motor by checking various parameters in the motor, such as voltage and current, to make sure the values are within a specific range. If the values are above or below the thresholds, the motor will coast and locked down to prevent any commands from being processed. 
```

Comment:
"lock down" not "locked down". Also, probably worth nothing that it "unlocks" once the parameters are back in the safe region